package.json

```diff
@@ -61,8 +61,7 @@
     "rollup-plugin-copy": "~3.4.0",
     "rollup-plugin-delete": "~1.2.0",
     "rollup-plugin-generate-html-template": "~1.7.0",
-    "rollup-plugin-postcss": "~2.0.3",
-    "rollup-plugin-scss": "~2.1.0",
+    "rollup-plugin-styles": "~3.14.1",
```

Comment:
This plugin says that it requires rollup 2, but as it is working too in x-components, and we should  upgrade rollup to its latest version soon I wouldn't bother downgrading it.